touchnet/bin/convert_hf_to_dcp.py

```
@@ -7,18 +7,28 @@
 import torch
 import torch.distributed.checkpoint as DCP
+import transformers
 from transformers import AutoModelForCausalLM
 from transformers.hf_argparser import HfArgumentParser
 
 import touchnet  # noqa
 from touchnet.bin import CkptConverterConfig
 from touchnet.utils.logging import init_logger, logger
 
+MODEL_DICT = {
+    "qwen2_audio": transformers.models.qwen2_audio.modeling_qwen2_audio.Qwen2AudioForConditionalGeneration,
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)